packages/sdk/tsup.config.js

```diff
@@ -0,0 +1,31 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig((options) => ({
+	entry: {
+		sdk: 'src/index.ts',
+	},
+	format: ['esm', 'cjs', 'iife'],
+	bundle: true,
+	dts: 'src/index.ts',
+	clean: true,
+	minify: !options.watch,
+	outExtension({ format }) {
+		let name = `.${format}.js`;
+
+		switch (format) {
+			case 'esm':
+				name = `.esm.min.js`;
+				break;
+			case 'cjs':
+				name = `.cjs.js`;
+				break;
+			case 'iife':
+				name = `.bundler.js`;
+				break;
```

Comment:
Should we update these to:

```js
index.mjs // esm
index.cjs // cjs
index.js  // agnostic iife ?
```

?

Reply:
I am not sure if all runtimes accept the `mjs` and `cjs` extensions 🤔 but if they do, definitely

Reply:
Ah yeah fair. My Node 20 defaultism privilege is showing :D